test/interpolations.jl

```diff
@@ -2,6 +2,22 @@ using DimensionalData
 using Interpolations
 
 f((x1, x2)) = log(x1+x2)
-A = f.(DimPoints((X(1:.1:10), Y(1:.5:20))))
+a = f.(DimPoints((X(1:1:10), Y(1:1.5:20))))
+b = f.(DimPoints((X(1:1:10), Y([1.0,3,7.5,10,15]))))
 to = rand(X(2:.3:7), Y(2:.3:17))
-out = DimensionalData.interp(A; to)
+# out = DimensionalData.interp(A; to)
+
+itp_a = linear_interpolation(a)
```

Comment:
We should test that we get the right answer too, with `@test`.  You can just define the ranges `2:0.3:7` etc as separate variables and use them both in defining the `DimArray` and in regular `linear_interpolation`

You'll need `using Test` at the top.